src/pages/HomePage/ListOfHouses/ListOfHouses.jsx

```diff
@@ -40,64 +86,28 @@ function ListOfHouses() {
           onDelete={handleDelete}
         />
       </Box>
-
       <Box component="div" className={styles.housesList}>
-        <Box component="div" className={styles.houseElement}>
-          <h4>420 Baker St, London</h4>
-          <p className={styles.price}>299,999£</p>
-          <img src="\src\assets\images\House.png" alt="House" />
-          <p className={styles.shortInfo}>2 bds 1 ba 1555 sqft - Apartament for sale</p>
-          <Button className={styles.moreInfo}>more info</Button>
-          <Checkbox
-            color="warning"
-            {...label}
-            icon={<FavoriteBorder />}
-            checkedIcon={<Favorite />}
-            className={styles.icon}
-          />
-        </Box>
-        <Box component="div" className={styles.houseElement}>
-          <h4>420 Baker St, London</h4>
-          <p className={styles.price}>299,999£</p>
-          <img src="\src\assets\images\House.png" alt="House" />
-          <p className={styles.shortInfo}>2 bds 1 ba 1555 sqft - Apartament for sale</p>
-          <Button className={styles.moreInfo}>more info</Button>
-          <Checkbox
-            color="warning"
-            {...label}
-            icon={<FavoriteBorder />}
-            checkedIcon={<Favorite />}
-            className={styles.icon}
-          />
-        </Box>
-        <Box component="div" className={styles.houseElement}>
-          <h4>420 Baker St, London</h4>
-          <p className={styles.price}>299,999£</p>
-          <img src="\src\assets\images\House.png" alt="House" />
-          <p className={styles.shortInfo}>2 bds 1 ba 1555 sqft - Apartament for sale</p>
-          <Button className={styles.moreInfo}>more info</Button>
-          <Checkbox
-            color="warning"
-            {...label}
-            icon={<FavoriteBorder />}
-            checkedIcon={<Favorite />}
-            className={styles.icon}
-          />
-        </Box>
-        <Box component="div" className={styles.houseElement}>
-          <h4>420 Baker St, London</h4>
-          <p className={styles.price}>299,999£</p>
-          <img src="\src\assets\images\House.png" alt="House" />
-          <p className={styles.shortInfo}>2 bds 1 ba 1555 sqft - Apartament for sale</p>
-          <Button className={styles.moreInfo}>more info</Button>
-          <Checkbox
-            color="warning"
-            {...label}
-            icon={<FavoriteBorder />}
-            checkedIcon={<Favorite />}
-            className={styles.icon}
-          />
-        </Box>
+        {data.map((item, i) => (
+          <Box component="div" className={styles.houseElement} key={sort[i][0].toString()}>
+            {/* Przez to że jest tyle renderowań nie moge sobie poradzić z ustawieniem Key. bo dając w mapie drugi arkument jaki np id jest ono array. Jakoś tak wykombinowałem ale nie wiem czy by tak mogło zostac */}
```

Comment:
Indexes of iterations of `map` in fact shouldn't be used as keys so it's really good that you used something different. However I'm not sure if this is going to work either because the keys are inconsistent between rerenders but I haven't found anything decisive that would prove or disprove what I'm writing. I'd advise using something from inside a rendered item, it could be a concatenation of 2 or more properties of a rendered objects as it gives a greater chance that it will stay unique if a single property could give this much confidence.